user/advanced-topics/usb-qubes.md

```diff
@@ -95,26 +102,67 @@ sys-usb dom0 allow
 
 (Change `sys-usb` to your desired USB qube.)
 
-You can now use your USB keyboard to login and for LUKS decryption during boot.
+You can now use your USB keyboard to log in and for LUKS decryption during
+boot.
 
-For a confirmation dialog each time the USB keyboard is connected, *which will effectively disable your USB keyboard for login and LUKS decryption*, change this line to:
+You can set up your system so that there's a confirmation prompt each time the
+USB keyboard is connected. However, this will effectively disable your USB
+keyboard for login and LUKS decryption, so **don't do this if you want to
+unlock your device with a USB keyboard!** If you're sure you wish to proceed,
+change the previous line to:
 
 ```
 sys-usb dom0 ask,default_target=dom0
 ```
 
-*Don't do that if you want to unlock your device with a USB keyboard!*
-
-Additionally, if you want to use USB keyboard to enter LUKS passphrase, it is incompatible with [hiding USB controllers from dom0](#how-to-hide-all-usb-controllers-from-dom0).
-You need to revert that procedure (remove `rd.qubes.hide_all_usb` option from files mentioned there) and employ alternative protection during system boot - disconnect other devices during startup.
-
-## Auto Enabling A USB Mouse ##
-
-**Caution:** Please carefully read the [Security Warning about USB Input Devices](/doc/device-handling-security/#security-warning-on-usb-input-devices) before proceeding.
-
-Handling a USB mouse isn't as critical as handling a keyboard, since you can login using the keyboard and accept the popup dialogue using your keyboard alone.
-
-If you want to attach the USB mouse automatically anyway, you have to edit the `qubes.InputMouse` policy file in dom0, located at:
+If you wish to use a USB keyboard to enter your LUKS passphrase, you cannot
+[hide its USB controller from dom0](#how-to-hide-usb-controllers-from-dom0). If
+you've already hidden that USB controller from dom0, you must revert the
+procedure by removing the `rd.qubes.hide_all_usb` option and employ an
+alternative strategy for protecting your system by physically disconnecting
+other devices during startup. You should also add the
+`usbcore.authorized_default=0` option, which prevents the initialization of
+non-input devices. (Qubes ships with a USBGuard configuration that allows only
+input devices when `usbcore.authorized_default=0` is set.)
+
+
+### How to enable a USB keyboard on a separate USB controller
```

Comment:
This section is R4.1-only. You can achieve subset of it manually on R4.0, but it's a bit clumsy so I don't think it's worth describing here.